src/client/components/pages/entities/identifiers.js

```diff
@@ -49,6 +53,7 @@ function EntityIdentifiers({identifierSet, identifierTypes}) {
 		</div>
 	);
 }
+
 EntityIdentifiers.displayName = 'EntityIdentifiers';
 EntityIdentifiers.propTypes = {
 	identifierSet: PropTypes.object.isRequired,
```

Comment:
I just realised now that I get an error in the web console when on a Publication page an Edition in the table does not have an identifierSet nor identifierTypes.

Here's for example a page that triggers it: http://localhost:9099/publication/de31f2e8-e29b-4190-87df-6c507a59c5b9


Simply remove `.isRequired` for both identifierSet and identifierTypes, and guard against a null identifierTypes as done on line 29


These are the errors I get. Nothing scary but an easy fix.

```
bundle.js:47684 Warning: Failed prop type: The prop `identifierSet` is marked as required in `EntityIdentifiers`, but its value is `undefined`.
in EntityIdentifiers (created by EntityLinks)
in EntityLinks (created by PublicationDisplayPage)
in div (created by PublicationDisplayPage)
in PublicationDisplayPage
in div (created by Layout)
in div (created by Layout)
in Layout
in e
printWarning @ bundle.js:47684
16:43:02.575 bundle.js:47684 Warning: Failed prop type: The prop `identifierTypes` is marked as required in `EntityIdentifiers`, but its value is `null`.
in EntityIdentifiers (created by EntityLinks)
in EntityLinks (created by PublicationDisplayPage)
in div (created by PublicationDisplayPage)
in PublicationDisplayPage
in div (created by Layout)
in div (created by Layout)
in Layout
in e
```